.github/workflows/image.yaml

```diff
@@ -43,3 +43,51 @@ jobs:
     secrets:
       registry_username: ${{ github.actor }}
       registry_password: ${{ secrets.CR_PAT }}
+
+  generate-init-kyverno-provenance:
+    needs: push-init-kyverno
+    permissions:
+      id-token: write # To sign the provenance.
+      packages: write # To upload assets to release.
+      actions: read #To read the workflow path.
+    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@main
+    with:
+      image: ghcr.io/${{ github.repository_owner }}/kyvernopre
+      digest: "${{ needs.push-init-kyverno.outputs.init_sha256_digest }}"
+      registry-username: ${{ github.actor }}
+    # TODO(https://github.com/slsa-framework/slsa-github-generator/issues/492): Remove after GA release.
+      compile-generator: true
+    secrets:
+      registry-password: ${{ secrets.CR_PAT }}
```

Comment:
This should not reference a manually-defined secret.

Reply:
I see this is already defined in the repo. Ignore.

Reply:
Can you please suggest a better way? We are already using CR_PAT in other places.

Reply:
Ignore my comment. I wasn't aware there was a custom secret in use currently. It's typically a GitHub embedded secret which is why I was confused.